Gopkg.toml

```diff
@@ -163,3 +163,7 @@ required = ["github.com/shirou/w32"]
 [[constraint]]
   branch = "master"
   name = "github.com/NYTimes/gziphandler"
+
+[[constraint]]
+  name = "github.com/atlassian/gostatsd"
+  version = "2.3.0"
```

Comment:
I just realized it looks like we are using an older version of gostatsd (latest is `5.4.0`). Any reason?

Reply:
I blame govendor! Haha yeah let me update that version 😉

Reply:
@palourde So this is why dep installed 2.3.0 and not most recent 🤔
```
Solving failure: No versions of github.com/atlassian/gostatsd met constraints:
	5.4.0: Could not introduce github.com/atlassian/gostatsd@5.4.0 due to a case-only variation: it depends on "github.com/sirupsen/logrus", but "github.com/Sirupsen/logrus" was already established as the case variant for that project root by the following other dependers:
	(root)
	github.com/docker/docker@v1.13.1
	github.com/tmc/grpc-websocket-proxy@timestamp_stdtime_example
```